website/content/docs/nia/cli/index.mdx

```diff
@@ -59,8 +59,25 @@ $ consul-terraform-sync -autocomplete-install
 When you start typing a CTS command, press the `<tab>` key to show a
 list of available completions. To show available flag completes, type `-<tab>`.
 
-If the `CTS_*` environment variables are set, the autocompletion
-automatically queries the running CTS server and returns helpful argument suggestions.
+Autocompletion will query the running CTS server to return helpful argument suggestions. For example, for the `task disable` command, autocompletion will return the names of all enabled tasks that can be disabled.
+
+When autocompletion makes the query to the running CTS server, it will also use any CTS_* environment variables (for example CTS_ADDRESS) set on the CTS server.
```

Comment:
I like the example here! Was wondering if you wanted to add the code formatting for CTS_* and CTS_ADDRESS e.g. `CTS_*` and `CTS_ADDRESS`?